numpy/__init__.pyi

```diff
@@ -452,7 +461,7 @@ class uint64(unsignedinteger):
     def __init__(self, __value: SupportsInt = ...) -> None: ...
 
 class inexact(number): ...  # type: ignore
-class floating(inexact, _real_generic): ...  # type: ignore
+class floating(inexact, _real_generic, float): ...  # type: ignore
```

Comment:
Don't you need to disable methods here too?

Reply:
The tricky thing here is that `np.float64` is an actual proper `float` subclass, _i.e._ disabling the methods will also disable it for the one subclass that _can_ use them. One way or another we'll end up with a few false positives or negatives.

Reply:
Can't you disable them here, and re-enable them on `np.float64`?

Reply:
Yup, done in https://github.com/numpy/numpy/pull/17117/commits/3a31a995a0a47fc821e3a8a21f712c1506f766ae.